components/ListPageDisplays/ListPageCards.stories.js

```diff
@@ -56,5 +56,14 @@ export const ArticleCards = () => (
         ],
       }}
     />
+    <ArticleCard
+      article={{
+        id: 'id3',
+        attachmentUrl: 'https://mockurl.jpeg',
```

Comment:
Suggest that we can put a placeholder image so that the image shows up in Storybook, which helps development & maintenance in the future.

```suggestion
        attachmentUrl: 'https://placekitten.com/512/1000',
```

![image](https://user-images.githubusercontent.com/108608/168966985-45f1cb4e-c7d6-44ca-9619-8ca8596e1b1f.png)